osu.Game.Rulesets.Osu/Mods/OsuModWiggle.cs

```diff
@@ -22,8 +24,21 @@ internal class OsuModWiggle : ModWithVisibilityAdjustment
         public override double ScoreMultiplier => 1;
         public override Type[] IncompatibleMods => new[] { typeof(OsuModTransform) };
 
-        private const int wiggle_duration = 90; // (ms) Higher = fewer wiggles
-        private const int wiggle_strength = 10; // Higher = stronger wiggles
+        [SettingSource("Strength")]
+        public BindableDouble WiggleStrength { get; } = new BindableDouble(1)
+        {
+            MinValue = .1f,
+            MaxValue = 2f,
+            Precision = .1f
```

Comment:
Generally leading zeroes are preferred when specifying floats.

```suggestion
            MinValue = 0.1f,
            MaxValue = 2f,
            Precision = 0.1f
```

Reply:
gotcha, will keep this in mind